### PR TITLE
Add marker function

### PR DESCRIPTION
**Pull request by ChamzasKonstantinos**
A helper addMarker function to enable adding externally created markers in the rviz helper

### CODE REVIEW

File: robowflex_library/src/io/visualization.cpp [hunk 1]
```diff
@@ -12,12 +12,12 @@
 
 #include <moveit/robot_state/conversions.h>
 
-#include <robowflex_library/log.h>
 #include <robowflex_library/builder.h>
 #include <robowflex_library/constants.h>
 #include <robowflex_library/geometry.h>
 #include <robowflex_library/io/colormap.h>
 #include <robowflex_library/io/visualization.h>
+#include <robowflex_library/log.h>
```
Comment (zkingston) [inline robowflex_library/src/io/visualization.cpp:20]: why did this header get shuffled?

File: robowflex_library/src/io/visualization.cpp [hunk 2]
```diff
@@ -377,6 +377,11 @@ void IO::RVIZHelper::removeMarker(const std::string &name)
         it->second.action = visualization_msgs::Marker::DELETE;
 }
 
+void IO::RVIZHelper::addMarker(const visualization_msgs::Marker &marker, const std::string &name)
```
Comment (zkingston) [inline robowflex_library/src/io/visualization.cpp:380]: There are many other addMarker functions - they should all call this one for purposes of code reuse. I believe they all end up creating a visualization_msgs::Marker in the end.

**Review by zkingston — CHANGES_REQUESTED** [review]
Looks good, just a small comment.

File: robowflex_library/src/io/visualization.cpp (hunk 1 above)
Comment (ChamzasKonstantinos) [reply robowflex_library/src/io/visualization.cpp]: Anyway I reverted it back

**Review by zkingston — APPROVED** [review]

**Pull request merged**